snowflake-resource-optimization/README.md

```diff
@@ -0,0 +1,72 @@
+# Snowflake Resource Optimization: Setup & Configuration
+
+<p align="center">
+<img src="./images/setup-config-demo.gif" alt="Incident Central demo gif" style="width: 600px;">
+</p>
+
+## Why you need this
+This Retool app extends [Snowflake's Resource Optimization: Setup & Configuration Quickstart Guide](https://quickstarts.snowflake.com/guide/resource_optimization_setup/index.html?index=..%2F..index#0).
+
+The Resource Optimization Quickstart Guides are meant to help customers better monitor and manage their credit consumption.
+
+Retool builds on the Quickstart Guide by enabling you to take action from right within the tool - set auto-suspend or auto-resume for your warehouses, suspend or drop idle warehouses, disable or drop idle users, etc.
+
+## What does Snowflake Resource Optimization: Setup & Configuration do?
+Snowflake Resource Optimization: Setup & Configuration is a ready-made app that lets you do these things:
```

Comment:
If true, would leave out the "ready-made" part in favor of playing up not having to type in commands.
E.g. something like

"The Snowflake Resource Optimization Dashboard lets you do the following things with the click of a button:"

Reply:
done